### ISSUE TITLE
Is it possible to have ranges of visible slots?

### ISSUE THREAD

**Issue by josehernandezv** [vkurko/calendar#213]
This is my use case, right now I am using slotMinTime and slotMaxTime and it works fine but I don't want to show that huge gap of empty slots in the middle, is there a way to hide those slots? A way to pass a range of slots instead of just one min and max or maybe hide those with CSS

<img width="798" alt="Screenshot 2023-12-27 at 10 51 22" src="https://github.com/vkurko/calendar/assets/28166305/1ae417a9-5361-4d9a-bcf7-29ff42f5cd41">

**Comment by vkurko** [comment]
Unfortunately, there is no such option right now. And it can't be done with CSS, because the event placement algorithm doesn't support this case. Do you know examples of something similar in other calendars?

**Comment by josehernandezv** [comment]
Not really, I was looking if it could be done with FullCalendar but I couldn't find an option there either.
Anyway, thank you for the quick reply and the awesome library.
